src/language-server/project/client.ts

```diff
@@ -330,10 +335,10 @@ export class GraphQLClientProject extends GraphQLProject {
       `Loading Apollo data for ${this.displayName}`,
       (async () => {
         try {
-          const { schemaTags, fieldStats } =
-            await engineClient.loadSchemaTagsAndFieldStats(serviceID);
+          const { schemaTags, fieldLatencies } =
+            await engineClient.loadSchemaTagsAndFieldLatencies(serviceID);
```

Comment:
I think we need to update the function being called as well

Reply:
picky, picky.
(forgot to save a file, whoops!)